.github/workflows/test.yml

```diff
@@ -13,12 +13,18 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    # set up conda
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+      uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: ${{ matrix.python-version }}
+        auto-activate-base: true
+        activate-environment: ""
+    # install rdkit and openbabel
+    - name: Install rdkit
+      run: conda create -c conda-forge -n my-rdkit-env python=${{ matrix.python-version }} rdkit openbabel;
```

Comment:
You may consider to install https://pypi.org/project/rdkit-pypi/, it should be faster than conda install. Although it is not an official package, we just use it to test the package.
See rdkit/rdkit#1812.

Reply:
I'll see this. But we still have dependency on obabel, which needs conda to install.